.github/workflows/publish-commit-bottles.yml

```
@@ -123,19 +120,20 @@ jobs:
           HOMEBREW_GITHUB_PACKAGES_USER: brewtestbot
           HOMEBREW_GITHUB_PACKAGES_TOKEN: ${{secrets.HOMEBREW_CORE_GITHUB_PACKAGES_TOKEN}}
         run: |
+          # Don't quote arguments that might be empty; this causes errors.
           brew pr-pull \
             --debug \
             --workflows=tests.yml \
             --committer="$BREWTESTBOT_NAME_EMAIL" \
             --root-url="https://ghcr.io/v2/homebrew/core" \
-            ${{inputs.autosquash && '' || '--no-autosquash'}} \
+            '${{inputs.autosquash && '--autosquash' || '--clean'}}' \
             ${{inputs.commit_bottles_to_pr_branch && '--no-cherry-pick' || ''}} \
-            ${{inputs.clean && '--clean' || ''}} \
-            ${{github.event.inputs.args}} \
-            ${{github.event.inputs.pull_request}}
+            ${{inputs.warn_on_upload_failure && '--warn-on-upload-failure' || ''}} \
+            ${{inputs.message && format('--message={0}', inputs.message) || ''}} \
```

```suggestion
            ${{inputs.message && format('--message="{0}"', inputs.message) || ''}} \
```

Should we quote the message or maybe the whole expression like with `--autosquash` above?

---

Quoting the whole expression will result in an error (I did that in an earlier commit). But quoting the message is a good suggestion; thanks.